views/index.pug

```diff
@@ -3,15 +3,59 @@ extends layout
 block content
   h1= title
   p Welcome to #{title}
+
+  a.i.fa.fa-pencil-square-o.createPostButton(href='#openModal1') Create Post
```

Comment:
I left some feedback about class names and CSS namespacing above, and in a previous PR.  Consider refactoring your CSS and HTML to make it more readable, and to provide more meaningful identifiers and organization.